lib/peep.rb

```diff
@@ -0,0 +1,47 @@
+require "pg"
+
+class Peep
+  attr_reader :id, :message, :date_posted, :keyword
+
+  def initialize(id:, message:, date_posted:)
+    @id = id
+    @message = message
+    @date_posted = date_posted
+  end
+
+  def self.all
+    if ENV["ENVIRONMENT"] == "test"
```

Comment:
You could use a one-liner here to improve your coverage and reduce the lines-per-method in your peep.rb file. You could also separate this into a method to DRY up your code (you can see what I mean in my code)